test/unit/StakingPool/setDescriptionIpfsHash.js

```diff
@@ -0,0 +1,64 @@
+const { ethers, expect } = require('hardhat');
+const { setEtherBalance } = require('../../utils/evm');
+
+describe('setDescriptionIpfsHash', function () {
```

Comment:
There's no function with the name setDescriptionIpfsHash

So i wouldn't call this like this

i would add a test in createStakingPool that verifies the event is emitted

it doesn't warrant a different file